lib/cell.rb

```diff
@@ -19,6 +19,7 @@ def empty?
 
   def place_ship(ship)
     @ship = ship
+    @empty = false
```

Comment:
I think I follow the logic here. We have updated our query's to access the cell's empty instance variable instead of accessing the ship nil value.